main.go

```diff
@@ -19,6 +19,7 @@ package main
 import (
 	"flag"
 	"os"
+	"sigs.k8s.io/jobset/pkg/util/cert"
```

Comment:
move this to the middle block of imports below

Reply:
done